test/e2e/cluster_upgrade.go

```diff
@@ -30,6 +30,7 @@ import (
 	"k8s.io/kubernetes/test/e2e/framework"
 	"k8s.io/kubernetes/test/e2e/framework/ginkgowrapper"
 	"k8s.io/kubernetes/test/e2e/upgrades"
+	apps "k8s.io/kubernetes/test/e2e/upgrades/apps"
```

Comment:
@foxish i believe i somehow modified BUILD files incorrectly. i have to put `apps` identifier. Otherwise, the compiler will treat `upgrades/apps` an import re-declaration of `upgrades`.